CHANGELOG.md

```diff
@@ -2,8 +2,8 @@
 - [v1.0.0 - v1.9.10](CHANGELOG-pre-v1.10.md)
 - [v0.11.6 and earlier](CHANGELOG-v0.md)
 
-## 1.18.0-rc1
-## September 18, 2024
+## 1.18.0 Enterprise
```

Comment:
Should this have `Enterprise`?

Reply:
it showed it in the example from the docs[here](https://hashicorp.atlassian.net/wiki/spaces/VAULT/pages/3303833604/1.18.0+GA+1.17.7+1.16.11+1.15.16#Creating-the-Changelog-PR-on-main), however when I look at 1.17.0 for example it does not have that so I'll remove.